projects/core/src/configurator/commons/store/actions/configurator-ui.action.ts

```diff
@@ -16,6 +18,17 @@ export class SetUiState implements StateEntityActions.EntityAction {
   }
 }
 
+export class SetCurrentGroup implements StateEntityActions.EntityAction {
+  readonly type = SET_CURRENT_GROUP;
+  readonly meta: StateEntityActions.EntityMeta;
+  constructor(productCode: string | string[], public payload: any) {
```

Comment:
payload in this case will always be type string, maybe change from any to string